frameworks/CSharp/aspnetcore/benchmark_config.json

```diff
@@ -20,9 +19,26 @@
       "notes": "",
       "versus": ""
     },
+    "middleware-json": {
+      "setup_file": "setup-json",
+      "json_url": "/json",
+      "port": 8080,
+      "approach": "Realistic",
+      "classification": "Micro",
+      "database": "None",
+      "framework": "aspnetcore",
+      "language": "C#",
+      "orm": "Raw",
+      "platform": "NET",
```

Comment:
"Mono" is not a platform; it is a language implementation which we either are or have implemented a fix for in the way of adding "flavor" to the `benchmark_config.json` files. In this case, the language would be "C#" and the flavor would be "Microsoft" (or something similar).

Unless "NET" and "NETCore" are distinct platforms for writing web applications, I do not see the need for the differentiation. It **sounds** like they are, and as such that **should** be reflected here.

I am not sure what "NETCoreAOT" is versus "NETCore".